styles/styles.css

```diff
@@ -0,0 +1,155 @@
+header {
+  background-image: url('../assets/hero.jpg');
+  /*background-position: 50% 50%;*/
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-bottom: solid;
+  border-color: #9D9494;
+  height: 100px;
+  color: #93737F;
+  font-family: 'Raleway', sans-serif;
+  text-align: center;
+  margin-top: 0;
+  padding: 100px;
+}
+
+header h1 {
+  margin-top: 15px;
+  font-size: 3em;
+}
+
+
+header nav {
+  padding: 0;
+  margin-top: 100px;
+}
+
+header nav li {
+  display: inline-block;
+  margin-left: 8px;
+  margin-right: 40px;
+}
+
+header nav li a {
+  color: #93737F;
+  text-decoration: none;
+  font-size: 1.3em;
+}
+
+a:hover {
+  color: black;
+}
+
+h2 {
+  text-align: center;
+}
+
+.circular--landscape {
+  display: inline-block;
+  position: relative;
+  width: 400px;
+  height: 400px;
+  overflow: hidden;
+  border-radius: 50%;
+}
+
+.portfolio nav li {
+  display: inline-block;
+}
+
+.portfolio li {
+  padding-left: 3%;
+}
+
+.portfolio ul li {
+  display: inline-block;
+  padding: 2%;
+  margin-left: 10%;
+}
+
+.portfolio ul li:nth-child(2) {
+  margin-left: 23%;
+  margin-right: 12%;
+}
+
+li {
+  list-style-type: none;
```

Comment:
This is my favorite CSS to use (okay, maybe second to `margin: 0 auto` for centering), be careful with it for when you _do_ want a list though! You could use a class for bullet-less lists if you ever run into that.